app/javascript/react/container/StateShowContainer.js

```diff
@@ -21,17 +23,31 @@ class StateShowContainer extends Component {
           }
         })
         .then(response => response.json())
-        .then(state => {
-          this.setState({ stateShow: state })
+        .then(stateHash => {
+          this.setState({ stateObject: stateHash.state, parks: stateHash.state.parks })
```

Comment:
If you look at your state after this setState, you'll see that it's 
```
stateObject: {
  ...
  parks: [
    ...
  ],
parks: [
  ...
]
```
So, the info is duplicated. Instead, it would be better to just accept the nesting and only have this.state.stateObject.

Reply:
Updated this